helix-term/src/ui/mod.rs

```diff
@@ -314,15 +314,20 @@ pub mod completers {
     }
 
     /// Recursive function to get all keys from this value and add them to vec
-    fn get_keys(value: &serde_json::Value, vec: &mut Vec<String>, scope: Option<&str>) {
+    fn get_keys(
+        value: &serde_json::Value,
+        vec: &mut Vec<String>,
+        scope: Option<&str>,
+        only_booleans: bool,
```

Comment:
i chose to defunctionalize here for simplicity purposes, but a predicate function could be passed as well.